src/screens/Customers/EditCustomer.tsx

```diff
@@ -49,6 +52,11 @@ function EditCustomer(props: EditCustomerProps) {
   const [meterNumber, setMeterNumber] = useState(currentCustomer.meterNumber);
   const [customerInactive, setCustomerInactive] = useState(!currentCustomer.isactive);
 
+  // Redirect to main customers page if no customer was passed
+  if (currentCustomer === EMPTY_CUSTOMER) {
```

Comment:
`EMPTY_CUSTOMER` was used here because the `useState` use `currentCustomer`. I also couldn't do this if clause before all the `useState` calls because any return statements need to happen after all the hooks execute (react yelled at me)